packages/iputils/iputils.spec

```
@@ -39,8 +38,6 @@ CONFIGURE_OPTS=(
  -DBUILD_NINFOD=false
  -DBUILD_RARPD=false
  -DBUILD_RDISC=false
- -DBUILD_TFTPD=false
- -DBUILD_TRACEROUTE6=false
```

I was curious why these cause trouble now, since they are/should be defined by iputils and the iputils build didn't change. All those options were already gone in the iputils build referenced here, but it wasn't until Meson 0.60 that passing unkonwn configure options causes build failures rather than warnings ([source](https://mesonbuild.com/Release-notes-for-0-60-0.html#unknown-options-are-now-always-fatal)).